examples/asr/speech_to_text_finetune.py

```diff
@@ -84,7 +84,23 @@ def get_base_model(cfg):
     elif nemo_model_path is not None:
         asr_model = ASRModel.restore_from(restore_path=nemo_model_path)
     elif pretrained_name is not None:
-        asr_model = ASRModel.from_pretrained(model_name=pretrained_name)
+        # Due to potential first time download of the model on the cluster, we need to make sure that only one
+        # rank downloads the model and the others wait for the download to finish.
+        num_ranks = trainer.num_devices * trainer.num_devices
+
+        if num_ranks > 1 and is_global_rank_zero():
+            asr_model = ASRModel.from_pretrained(model_name=pretrained_name)
+        else:
+            # Sleep on all ranks for at least 60 seconds
+            wait_time = int(cfg.get('exp_manager', {}).get('seconds_to_sleep', 60))
+            if wait_time < 60:
+                wait_time = 60
+                logging.info("Sleeping for at least 60 seconds to wait for model download to finish.")
```

Comment:
lets keep this out of if case and show the seconds if passed through cfg?

Reply:
Sounds good